ios/brave-ios/Sources/AIChat/Components/Messages/AIChatResponseMessageView.swift

```diff
@@ -94,6 +94,19 @@ struct AIChatResponseMessageView: View {
       AIChatMessageHeaderView(isUser: false, isEdited: lastEdited != nil)
 
       ForEach((turn.edits?.last ?? turn)?.events ?? [], id: \.self) { event in
+        if let images = turn.uploadedImages, !images.isEmpty {
+          ForEach(images, id: \.self) {
+            if let image = UIImage(from: $0) {
+              Image(uiImage: image)
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(width: 100.0, height: 100.0)
```

Comment:
This will make the layout appear to have gaps depending on the aspect ratio of the image. `contentMode: .fill` may be more appropriate, or remove explicit sizes

Reply:
Removed entirely. It's on another branch now